### PR TITLE
CHG-12098 chmod cleanup

### PR DESCRIPTION
**Pull request by ThrawnCA**
Only run chown/chmod on storage directories once.

### CODE REVIEW

File: recipes/ckan-configure.rb [hunk 1]
```diff
@@ -41,3 +41,11 @@
 execute "Update supervisor workers if needed" do
     command "supervisorctl update"
 end
+
+if not system("grep 'alias git=' ~/.bash_profile")
+    execute "Add CKAN Git alias to Bash" do
+        command <<-EOS
+            echo "alias git='sudo -u ckan git'" >> ~/.bash_profile
```
Comment (duttonw) [inline recipes/ckan-configure.rb:48]: what is this doing?

File: recipes/ckanweb-deploy.rb [hunk 2]
```diff
@@ -52,13 +52,14 @@
...
-	execute "Ensure files in #{nfs_path} have correct ownership" do
-		command "chown -R #{dir_owner}:#{service_name} #{nfs_path}"
-	end
+end
 
-	execute "Ensure files in #{nfs_path} have correct permissions" do
-		command "chmod -R g+rwX #{nfs_path}"
-	end
+execute "Ensure files in storage have correct ownership" do
+	command "chown -R #{service_name}:#{service_name} #{storage_root}"
```
Comment (duttonw) [inline recipes/ckanweb-deploy.rb:58]: can you nohup & this command

**Review by duttonw — APPROVED** [review]

File: recipes/ckan-configure.rb (hunk 1 above)
Comment (ThrawnCA) [reply recipes/ckan-configure.rb:48]: Amazon Linux 2 is stricter about Git security; even read-only Git operations can't be performed in a directory owned by something else. So this is aliasing `git` to `sudo -u ckan git` to make it easier to work with repositories owned by `ckan`.

File: recipes/ckanweb-deploy.rb [hunk 3]
```diff
@@ -52,13 +52,14 @@
...
+end
+
+execute "Ensure files in storage have correct permissions" do
+	command "chmod -R g+rwX #{storage_root}"
```
[the remark this answers is not in the document]
Comment (ThrawnCA) [reply recipes/ckanweb-deploy.rb:62]: Blocking is the right behaviour here, to avoid permission errors.

**Pull request merged**